data/src/main/java/com/teamagam/gimelgimel/data/message/repository/cache/room/entities/AlertFeatureEntity.java

```diff
@@ -0,0 +1,10 @@
+package com.teamagam.gimelgimel.data.message.repository.cache.room.entities;
+
+public class AlertFeatureEntity {
```

Comment:
I see that an `@Entity` annotation is not needed here, if I understand how things work it is because we use a converter and we don't relate a new table to those entities.
Since so, can't we use our already existing entities? For example, this class and the `Alert` class share literally the same fields.